validation-test/Sanitizers/tsan-inout.swift

```diff
@@ -2,18 +2,23 @@
 // RUN: cd %t
 // RUN: %target-build-swift %S/Inputs/tsan-uninstrumented.swift -target %sanitizers-target-triple -module-name TSanUninstrumented -emit-module -emit-module-path %t/TSanUninstrumented.swiftmodule -parse-as-library
 // RUN: %target-build-swift %S/Inputs/tsan-uninstrumented.swift -target %sanitizers-target-triple -c -module-name TSanUninstrumented -parse-as-library -o %t/TSanUninstrumented.o
-// RUN: %target-swiftc_driver %s %t/TSanUninstrumented.o -target %sanitizers-target-triple -I%t -L%t -g -sanitize=thread -o %t/tsan-binary
+// RUN: %target-swiftc_driver %s %t/TSanUninstrumented.o -target %sanitizers-target-triple -I%t -L%t -g -sanitize=thread %import-libdispatch -o %t/tsan-binary
 // RUN: not env %env-TSAN_OPTIONS=abort_on_error=0 %target-run %t/tsan-binary 2>&1 | %FileCheck %s
 // RUN: not env %env-TSAN_OPTIONS=abort_on_error=0:ignore_interceptors_accesses=0 %target-run %t/tsan-binary 2>&1 | %FileCheck %s --check-prefix CHECK-INTERCEPTORS-ACCESSES
 // REQUIRES: executable_test
 // REQUIRES: stress_test
-// REQUIRES: objc_interop
 // REQUIRES: tsan_runtime
 
 // Test ThreadSanitizer execution end-to-end when calling
 // an uninstrumented module with inout parameters
 
-import Darwin
+#if canImport(Darwin)
+  import Darwin
+#elseif canImport(Glibc)
+  import Glibc
+#else
+#error("Unsupported platform")
```

Comment:
This doesn't account for Windows

Reply:
What is the general state of TSan for Swift on Windows? I was unaware that TSan tests ran at all under Windows.

Reply:
Well, all the tests run on Windows, and this is changing the fact that the test should fail to should succeed, but it wont actually do so on Windows.  Marking the TSAN tests as XFAIL on `OS=windows-msvc` is reasonable.

Reply:
I *think* that `// REQUIRES: tsan_runtime` makes this test not run on Windows (since we don't produce a TSan runtime there). If we start to support TSan on Windows, we have to adapt this test, but I consider this out of scope for this PR.